Dockerfile

```diff
@@ -173,9 +173,24 @@ COPY --from=centos8-build-diagnostics /work/cache /work/cache
 COPY --from=centos8-build-wrapper /work/cache /work/cache
 RUN ./pkg/rpm/build.sh
 
+
+FROM centos8-build AS centos8-build-plugin
+WORKDIR /work
+COPY cmd/ops_agent_uap_plugin cmd/ops_agent_uap_plugin
+COPY ./builds/ops_agent_plugin.sh .
+RUN ./ops_agent_plugin.sh /work/cache/
+
+
+FROM centos8-build AS centos8-package-plugin
```

Comment:
Is it necessary to have a separate image for packaging? Can we use the same `centos8-build-plugin` image for both building and packaging?

Reply:
Done